fabbwled-backend/src/main/resources/sections/001-036.yaml

```diff
@@ -0,0 +1,16 @@
+- number: 15
+  content:
+    - text: Three drunken army officers...
+    - if:
+        hasTitle: "Protector of Sokara"
+      then:
+        - turnTo: 542
+      else:
+        - text: If not, read on.
+        - choice:
+            - text: Step out of the way
+              then:
+                - turnTo: 44
+            - text: Return the insult
+              then:
+                - turnTo: 266
```

Comment:
```suggestion

```

Reply:
oops, theres a test for this section. reverting.